FBSDKMessengerShareKit/FBSDKMessengerShareKit.xcodeproj/project.pbxproj

```diff
@@ -534,41 +534,49 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 9CAE101E1B15355200163BDA /* FacebookSDK-LogicTests.xcconfig */;
 			buildSettings = {
+				PRODUCT_BUNDLE_IDENTIFIER = "com.facebook.sdk.$(PRODUCT_NAME:rfc1034identifier)";
 			};
 			name = Debug;
 		};
 		9CAE101C1B15353900163BDA /* Release */ = {
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 9CAE101E1B15355200163BDA /* FacebookSDK-LogicTests.xcconfig */;
 			buildSettings = {
+				PRODUCT_BUNDLE_IDENTIFIER = "com.facebook.sdk.$(PRODUCT_NAME:rfc1034identifier)";
 			};
 			name = Release;
 		};
 		9D0558B71A797E7B009B7108 /* Debug */ = {
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 23AF544A1A8AB66C001D9871 /* FacebookSDK-Project-Debug.xcconfig */;
 			buildSettings = {
+				ENABLE_TESTABILITY = YES;
+				GCC_NO_COMMON_BLOCKS = YES;
+				ONLY_ACTIVE_ARCH = YES;
 			};
 			name = Debug;
 		};
 		9D0558B81A797E7B009B7108 /* Release */ = {
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 23AF544B1A8AB66C001D9871 /* FacebookSDK-Project-Release.xcconfig */;
 			buildSettings = {
+				GCC_NO_COMMON_BLOCKS = YES;
 			};
 			name = Release;
 		};
 		9D0558BA1A797E7B009B7108 /* Debug */ = {
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 23AF54481A8AB66C001D9871 /* FacebookSDK-Library.xcconfig */;
 			buildSettings = {
+				PRODUCT_BUNDLE_IDENTIFIER = "com.facebook.sdk.$(PRODUCT_NAME:rfc1034identifier)";
```

Comment:
Also can be moved to FBSDKMessengerShareKit/Configurations/FacebookSDK-Project.xcconfig